cinema/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. The correct syntax should be `from rest_framework.permissions import BasePermission, SAFE_METHODS`. The `/` character is not valid here and should be removed.